src/main/java/org/gradle/wrapperupgrade/UpgradeWrapper.java

```diff
@@ -113,34 +108,34 @@ private boolean isWrapperChanged(Path gitCheckoutDir) {
         }
     }
 
-    private void createPr(Params params, String usedBuildToolVersion) throws IOException {
-        String shortDesc = createShortDescription(params, usedBuildToolVersion);
-        String longDesc = createLongDescription(params, usedBuildToolVersion);
+    private void createPr(Params params) throws IOException {
+        String shortDesc = createShortDescription(params);
+        String longDesc = createLongDescription(params);
         gitCommitAndPush(params, shortDesc);
         gitCreatePr(params, shortDesc, longDesc);
     }
 
-    private String createShortDescription(Params params, String usedBuildToolVersion) {
+    private String createShortDescription(Params params) {
         String buildToolName = buildToolStrategy.buildToolName();
         String latestBuildToolVersion = params.latestBuildToolVersion.version;
         String relativePath = params.rootProjectDirRelativePath.normalize().toString();
 
         StringBuilder description = new StringBuilder();
-        description.append(String.format("Bump %s Wrapper from %s to %s", buildToolName, usedBuildToolVersion, latestBuildToolVersion));
+        description.append(String.format("Bump %s Wrapper from %s to %s", buildToolName, params.usedBuildToolVersion.version, latestBuildToolVersion));
```

Comment:
For consistency, and a shorter method invocation, extract `params.usedBuildToolVersion.version` into a local variable, like done for `latestBuildToolVersion`.

Reply:
Did you apply this suggestion?

Reply:
ah I forgot to apply it in both methods, now it's there